### DISCUSSION TITLE
French translation

### DISCUSSION THREAD

**Discussion by Chami001** [wlcrs/huawei_solar#688] [General]
Hi all,

Firstly, thank you and congratulations for this excellent app.

I'm a new user from Switzerland and I speak French. I've created a file named fr.json for my family's use. If you'd like to integrate it into the branch, I send it as an attached file.

Thanks again and have a good day.

[fr.json](https://github.com/wlcrs/huawei_solar/files/15020606/fr.json)

**Comment by wlcrs** [comment]
Merci beaucoup!

I've added your translation file to the repository. It will be included in the next release.

For future updates, feel free to create a pull request. If you are not familiar with how to do that, I would recommend using https://github.dev/wlcrs/huawei_solar : this gives you an online editor which makes it easy to change files and create a GitHub pull request with the changes.
